common/djangoapps/student/views.py

```diff
@@ -305,35 +304,41 @@ def get_course_enrollments(user, orgs_to_include, orgs_to_exclude):
             )
             continue
 
-        # Filter out anything that is not attributed to the orgs to include.
-        if orgs_to_include and course_overview.location.org not in orgs_to_include:
+        # Filter out anything that is not in the whitelist.
+        if org_whitelist and course_overview.location.org not in org_whitelist:
             continue
 
-        # Conversely, filter out any enrollments with courses attributed to current ORG.
-        elif course_overview.location.org in orgs_to_exclude:
+        # Conversely, filter out any enrollments in the blacklist.
+        elif org_blacklist and course_overview.location.org in org_blacklist:
             continue
 
         # Else, include the enrollment.
         else:
             yield enrollment
 
 
-def get_course_certificates(user):
+def get_site_course_enrollments(user):
     """
-    Returns information about a user's course certificates.
+    Returns a user's enrollments filtered by the orgs of the current site.
```

Comment:
Reviewers:
1. I refactored this function to make it more clear.
2. I had issues with my devstack, so I have not yet run this code.
3. It is possible that this whitelist/blacklist code introduced a bug, but if we can get it right, it is a lot easier to understand.